Public/Src/Utilities/Utilities/VmCommandProxy/VmInitializer.cs

```diff
@@ -25,7 +25,7 @@ public class VmInitializer
         /// <remarks>
         /// According to CB team, VM initialization roughly takes 2-3 minutes.
         /// </remarks>
-        private const int InitVmTimeoutInMinute = 10;
+        private const int InitVmTimeoutInMinute = 15;

```

Comment:
Is this related?  It looks to me that it is not

Reply:
Good catch, search replace issue.